doc/users/explain/figures.rst

```diff
@@ -0,0 +1,215 @@
+
+.. _figure_explanation:
+
+================================================
+Creating, viewing, and saving Matplotlib Figures
+================================================
+
+.. plot::
+    :include-source:
+
+    fig = plt.figure(figsize=(2, 2), facecolor='lightskyblue',
+                     layout='constrained')
+    fig.suptitle('Figure')
+    ax = fig.add_subplot()
+    ax.set_title('Axes', loc='left', fontstyle='oblique', fontsize='medium')
+
+When looking at Matplotlib visualization, you are almost always looking at
+Artists placed on a `~.Figure`.  In the example above, the figure is the
+blue region and `~.Figure.add_subplot` has added an `~.axes.Axes` artist to the
+`~.Figure` (see :ref:`figure_parts`).  A more complicated visualization can add
+multiple Axes to the Figure, colorbars, legends, annotations, and the Axes
+themselves can have multiple Artists added to them
+(e.g. ``ax.plot`` or ``ax.imshow``).
+
+.. _viewing_figures:
+
+Viewing Figures
+================
+
+We will discuss how to create Figures in more detail below, but first it is
+helpful to understand how to view a Figure.  This varies based on how you are
+using Matplotlib, and what :ref:`Backend <what-is-a-backend>` you are using.
+
+Notebooks and IDEs
+------------------
+
+If you are using a Notebook (e.g. `Jupyter <https://jupyter.org>`_) or an IDE
+that renders Notebooks (PyCharm, VSCode, etc), then they have a backend that
+will render the Matplotlib Figure when a code cell is executed.  One thing to
+be aware of is that the default Jupyter backend (``%matplotlib inline``) will
+by default trim or expand the figure size to have a tight box around Artists
+added to the Figure (see :ref:`saving_figures`, below).
+
+Standalone scripts and interctive use
+-------------------------------------
+
+If the user is on a client with a windowing system, there are a number of
+:ref:`Backends <what-is-a-backend>` that can be used to render the Figure to
+the screen, usually using a Python Qt, Tk, or Wx toolkit, though there is a native
+MacOS backend as well.  These are typically chosen either in the user's
+:ref:`matplotlibrc <customizing-with-matplotlibrc-files>`, or by calling
+``matplotlib.use('QtAgg')`` at the beginning of a session or script.
+
+When run from a script, or interactively (e.g. from an
+`iPython shell <https://https://ipython.readthedocs.io/en/stable/>`_) the Figure
+will not be shown until we call ``plt.show()``. The Figure will appear in
+a new GUI window, and usually will have a toolbar with Zoom, Pan, and other tools
+for interacting with the Figure.  By default, ``plt.show()`` blocks
+further interaction from the script or shell until the Figure window is closed,
+though that can be toggled off for some pusposes.  For more details, please see
```

Comment:
```suggestion
though that can be toggled off for some purposes.  For more details, please see
```

Reply:
Thanks, for some reason I had my spellchecker off, but now its on again ;-)